tests/v2/validation/provisioning/ssh.go

```diff
@@ -0,0 +1,34 @@
+package provisioning
+
+// This file contains all tests that require to ssh into a node to run commands to check things
+// such as any stats, benchmarks, etc. For example, ssh is required to check the cpu usage of a
+// process running on an individual node.
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/rancher/rancher/tests/framework/pkg/nodes"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	cpuUsageVar = 100
+)
+
+// This func checks the cpu usage of the cluster agent. If the usage is too high the func will return a warning.
+func CheckCPU(node *nodes.Node) (string, error) {
+	command := "ps -C agent -o %cpu --no-header"
```

Comment:
this will fail on windows nodes. But since this is only for node driver tests, it should be OK for now since we don't have windows in vsphere automation yet. We should fix this when that is added though